local/postman/TX-Training.postman_collection.json

```diff
@@ -0,0 +1,1851 @@
+									"pm.collectionVariables.set(\"PULLENDPOINT_URL\", pullEndpointUrl)",
+									"var pullAuthToken = pm.response.json()[\"authorization\"]",
+									"pm.collectionVariables.set(\"PULL_AUTHTOKEN\", pullAuthToken)"
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "x-api-key",
+								"value": "TESTUSERPASSWORD_123",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "http://customer-control-plane:8181/management/v2/edrs/{{TRANSFER_ID}}/dataaddress",
+							"protocol": "http",
+							"host": [
+								"customer-control-plane"
+							],
+							"port": "8181",
+							"path": [
+								"management",
+								"v2",
+								"edrs",
+								"{{TRANSFER_ID}}",
+								"dataaddress"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "DTR Lookup via EDC PULL",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									""
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						},
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									""
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "{{PULL_AUTHTOKEN}}",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{PULLENDPOINT_URL}}/api/v3/lookup/shells?assetIds={{ENCODED_QUERY}}",
+							"host": [
+								"{{PULLENDPOINT_URL}}"
+							],
+							"path": [
+								"api",
+								"v3",
+								"lookup",
+								"shells"
+							],
+							"query": [
+								{
+									"key": "assetIds",
+									"value": "{{ENCODED_QUERY}}"
+								}
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "DTR Specific Query via EDC PULL",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									""
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						},
+						{
+							"listen": "prerequest",
+							"script": {
+								"exec": [
+									""
+								],
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "{{PULL_AUTHTOKEN}}",
+								"type": "text"
+							}
+						],
+						"url": {
+							"raw": "{{PULLENDPOINT_URL}}/api/v3/shell-descriptors/{{ENCODED_SHELL_ID}}",
+							"host": [
+								"{{PULLENDPOINT_URL}}"
+							],
+							"path": [
+								"api",
+								"v3",
+								"shell-descriptors",
+								"{{ENCODED_SHELL_ID}}"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Step 4",
```

Comment:
The other step 4 is missing doing also the submodel request against the puris submodel endpoint is missing. Is that on purpose? We could remain as currently but the doing the call here the first time would also be good for the trainees.

Reply:
I'm not quite sure I understand how that is meant. Maybe we could have short talk about it?

Reply:
As discussed: Let's insert a step to also do the submodel query using the puris api and elaborate a little how this works so that participants understand it easily